src/pages/index.tsx

```diff
@@ -24,6 +25,12 @@ const App = () => {
       <FeedBack />
     </Layout>
   );
+
+  return (
+    <Router>
+      <Home path="/" />
```

Comment:
why is this required?
gatsby automatically puts index into `/` path

Reply:
Removed, I was trying something and then didn't need it !